docs/data-guidelines.md

```diff
@@ -56,13 +56,15 @@ For example, the feature for a `focus` event targeting the `Element` interface w
 }
 ```
 
-This rule applies to the event features themselves, not the features for the event handlers. For example, `focus_event` and `onfocus` are two separate features.
+The event handler `onfocus` is included in the `focus_event` entry. If an implementation doesn't support the event handler property, use `partial_implementation`.
```

Comment:
Can you suggest what the note text should be?

Can you also say what should happen if `onfocus` is supported but the "focus" event cannot be fired? That can mess with feature detection, but I think we should probably still just not record it in BCD.

Reply:
Suggested a note text